src/bp/ui-studio/src/web/views/FlowBuilder/nodeProps/SelectActionDropdown.jsx

```diff
@@ -37,6 +37,7 @@ export default class SelectActionDropdown extends Component {
         onChange={this.props.onChange}
         options={this.props.options}
         value={this.props.value}
+        isClearable={this.props.onClear}
```

Comment:
isClearable is a bool, can you add an explicit cast

Reply:
Sets the isClearable value to the method which sets the selected action to undefined. The behaviour isn't clear

Reply:
🤣 wow yeah I did that